dbt-snowflake/src/dbt/include/snowflake/macros/relations/table/create/standard.sql

```diff
@@ -0,0 +1,55 @@
+{% macro snowflake__create_table_standard_sql(relation, compiled_code) -%}
+{#-
+    Implements CREATE TABLE and CREATE TABLE ... AS SELECT:
+    https://docs.snowflake.com/en/sql-reference/sql/create-table
+    https://docs.snowflake.com/en/sql-reference/sql/create-table#create-table-as-select-also-referred-to-as-ctas
+-#}
+
+{%- set catalog_relation = adapter.build_catalog_relation(config.model) -%}
+
+{%- if catalog_relation.is_transient -%}
+    {%- set transient='transient ' -%}
+{%- else -%}
+    {%- set transient='' -%}
+{%- endif -%}
+
+{%- set copy_grants = config.get('copy_grants', default=false) -%}
+
+{%- set contract_config = config.get('contract') -%}
+{%- if contract_config.enforced -%}
+    {{- get_assert_columns_equivalent(compiled_code) -}}
+    {%- set compiled_code = get_select_subquery(compiled_code) -%}
+{%- endif -%}
+
+{%- set sql_header = config.get('sql_header', none) -%}
+{{ sql_header if sql_header is not none }}
+
+create or replace {{ transient }}table {{ relation }}
```

Comment:
nit: WS

Reply:
Are you referring to the lack of a space between `{{ transient }}` and `table`? I did this because the resulting sql will be spaced correctly. If this is not a transient table, then this should be:
`create or replace table <my_table>...`
If this is a transient table, then this should be:
`create or replace transient table <my_table>...`
The difference is `transient ` (with the trailing space. You could also do it as the leading space. Either way, this code looks a little wonky so that the generated code is spaced correctly.

If you're referring to something else, then I'm lost.